### ISSUE TITLE
The misclassified instances in waterbirds

### ISSUE THREAD

**Issue by xieyxclack** [anniesch/jtt#6]
Many thanks for such great work!

I am trying to reproduce the reported experimental results, but I found that for the Waterbirds dataset, when I train the initialization ERM model for T=50 epochs, the training accuracy is 100%, so the error set is empty.  I follow the suggested models and hyper-parameters: ResNet-50 with pretrained on imagenet, lr=1e-3, and weight_decay=1e-4.

Do you have any idea about the phenomenon? Do I misunderstand the proposed methodology or the experimental setting?
Thanks again!

**Comment by anniesch** [comment]
Hi @xieyxclack, the suggested hyperparameters for JTT on Waterbirds are lr=1e-5 and weight_decay=1.0 for _both stages_ of training. Hope that helps! Let me know if you have more questions.
